chcemvediet/apps/inforequests/tests/test_models/test_deadline.py

```diff
@@ -0,0 +1,157 @@
+            self.assertEqual(deadline.is_in_workdays, is_in_workdays)
+
+    def test_deadline_date_property(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-10'), value=8, unit=Deadline.UNITS.CALENDAR_DAYS)
+        self.assertEqual(deadline.deadline_date, naive_date(u'2010-10-18'))
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-10'), value=8, unit=Deadline.UNITS.WORKDAYS)
+        self.assertEqual(deadline.deadline_date, naive_date(u'2010-10-20'))
+
+    def test_calendar_days_passed_property_and_calendar_days_passed_at_method(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-05'), _today=naive_date(u'2010-10-10'))
+        self.assertEqual(deadline.calendar_days_passed, 5)
+        self.assertEqual(deadline.calendar_days_passed_at(naive_date(u'2010-10-10')), 5)
+
+    def test_workdays_passed_property_and_workdays_passed_at_method(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-05'), _today=naive_date(u'2010-10-10'))
+        self.assertEqual(deadline.workdays_passed, 3)
+        self.assertEqual(deadline.workdays_passed_at(naive_date(u'2010-10-10')), 3)
+
+    def test_calendar_days_remaining_property_and_calendar_days_remaining_at_method(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-05'), value=8, unit=Deadline.UNITS.CALENDAR_DAYS, _today=naive_date(u'2010-10-08'))
+        self.assertEqual(deadline.calendar_days_remaining, 5)
+        self.assertEqual(deadline.calendar_days_remaining_at(naive_date(u'2010-10-08')), 5)
+
+    def test_workdays_remaining_property_and_workdays_remaining_at_method(self):
+        deadline =  self._create_deadline(base_date=naive_date(u'2010-10-05'), value=8, unit=Deadline.UNITS.CALENDAR_DAYS, _today=naive_date(u'2010-10-08'))
+        self.assertEqual(deadline.workdays_remaining, 3)
+        self.assertEqual(deadline.workdays_remaining_at(naive_date(u'2010-10-08')), 3)
+
+    def test_calendar_days_behind_property_and_calendar_days_behind_at_method(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-05'), value=8, unit=Deadline.UNITS.CALENDAR_DAYS, _today=naive_date(u'2010-10-26'))
+        self.assertEqual(deadline.calendar_days_behind, 13)
+        self.assertEqual(deadline.calendar_days_behind_at(naive_date(u'2010-10-26')), 13)
+
+    def test_workdays_behind_property_and_workdays_behind_at_method(self):
+        deadline = self._create_deadline(base_date=naive_date(u'2010-10-05'), value=8, unit=Deadline.UNITS.CALENDAR_DAYS, _today=naive_date(u'2010-10-26'))
+        self.assertEqual(deadline.workdays_behind, 9)
+        self.assertEqual(deadline.workdays_behind_at(naive_date(u'2010-10-26')), 9)
+
+    def test_is_deadline_missed_property_and_is_deadline_missed_at_method(self):
+        tests = (                            # _today,                    is_deadline_missed
```

Comment:
@viliambalaz Treba tieto komentare pri stlcoch? Ved je to zjavne z `for unit, _today, is_deadline_missed` nizsie.

Reply:
:ok:

Reply:
@viliambalaz Este ostali v niektorych testoch

Reply:
Ok

Reply:
Upravim to v review commite uz.